index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="top-socials">
+                <img src="images/facebook.png" alt="facebook" class="top-img">
+                <img src="images/twitter.png" alt="twitter" class="top-img">
```

Comment:
[OPTIONAL]
- [ ] It would be nicer to grap these socials images inside an anchor tag.
```suggestion
               <a href="https://www.facebook.com"> <img src="images/facebook.png" alt="facebook" class="top-img"></a>
                <a href="https://www..."><img src="images/twitter.png" alt="twitter" class="top-img"></a>
```